moviecastcrew/__init__.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python
```

Comment:
@akiva17 This is a basic package structure of python. It lets you group imports for objects and functions.

See the python docs here:
https://packaging.python.org/tutorials/packaging-projects/